experiments/veo-app/pages/library.py

```diff
@@ -35,11 +35,16 @@
 )
 from components.header import header
 from components.library.image_details import CarouselState, image_details
+from components.library.video_details import video_details
```

Comment:
I'm thinking that we should move most of the imports to the `__all__` at the root `__init__.py` but can be done later. The imports are getting bigger